recipes/safe-pysha3/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "safe-pysha3" %}
+{% set version = "1.0.4" %}
+{% set commit = "54be7e36b56c4734ca358c26b7734516cd748189" %}
+{% set sha256 = "292548b0e18d6043a468185d6ce27761204ebf4d694daef956b6ffb52a6871a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/5afe/pysha3/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vvv
+  number: 0
+  skip: true  # [py<=3.8]
+  missing_dso_whitelist:
```

Comment:
This should not be needed.

Reply:
It seems to be needed:
```
number of files: 11
  ERROR (safe-pysha3,lib/python3.8/site-packages/_pysha3.cpython-38-x86_64-linux-gnu.so): /lib64/libc.so.6 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
```
Does the recipe needs a `build` section?